src/main/java/org/fundacionjala/coding/kevinherrera/crypto/IVigenereCypher.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.coding.kevinherrera.crypto;
+
+/**
+ * Interface for vigenere cypher.
+ */
+public interface IVigenereCypher {

```

Comment:
this class is redundant.
Please move the abstract methods (encode, decode) inside CesarCrypto class